Crash playing media

### Describe the bug

1. Start Jellyfin, set the bitrate to 120Mbps to avoid transcoding
2. Attempting to play a specific video file results in an app crash. This video file works fine with transcoding. 

I can post media info if it would help, but naively looking at the logs it looks like we're trying to call into VLC whilst Exoplayer is playing. 

### Logs

---

client: Jellyfin for Android TV
client_version: 0.15.0-beta.2
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown

---
### Logs

***Stack Trace***: 
```log
java.lang.IllegalStateException: can't get VLCObject instance
	at org.videolan.libvlc.MediaPlayer.nativeSetAudioOutputDevice(Native Method)
	at org.videolan.libvlc.MediaPlayer.setAudioOutputDeviceInternal(MediaPlayer.java:995)
	at org.videolan.libvlc.MediaPlayer.updateAudioOutputDevice(MediaPlayer.java:485)
	at org.videolan.libvlc.MediaPlayer.access$600(MediaPlayer.java:58)
	at org.videolan.libvlc.MediaPlayer$3.onAudioDevicesChanged(MediaPlayer.java:561)
	at org.videolan.libvlc.MediaPlayer$3.onAudioDevicesRemoved(MediaPlayer.java:585)
	at android.media.AudioManager$NativeEventHandlerDelegate$1.handleMessage(AudioManager.java:6270)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7668)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

```  
***Logcat***: 
```log
--------- beginning of main
11-21 20:03:31.335 D/VideoManager(28020): Caught player timeline change - reason: SOURCE_UPDATE
11-21 20:03:31.407 I/jellyfin-apiclient(28020): Response received from: https://<domain>/Sessions/Playing
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020): Playback error
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):   com.google.android.exoplayer2.ExoPlaybackException: Source error
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.ExoPlayerImplInternal.handleIoException(ExoPlayerImplInternal.java:632)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.ExoPlayerImplInternal.handleMessage(ExoPlayerImplInternal.java:604)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at android.os.Handler.dispatchMessage(Handler.java:102)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at android.os.Looper.loop(Looper.java:223)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at android.os.HandlerThread.run(HandlerThread.java:67)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):   Caused by: com.google.android.exoplayer2.upstream.HttpDataSource$InvalidResponseCodeException: Response code: 416
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.open(DefaultHttpDataSource.java:404)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.upstream.DefaultDataSource.open(DefaultDataSource.java:258)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.upstream.StatsDataSource.open(StatsDataSource.java:84)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.source.ProgressiveMediaPeriod$ExtractingLoadable.load(ProgressiveMediaPeriod.java:1005)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.upstream.Loader$LoadTask.run(Loader.java:412)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at java.lang.Thread.run(Thread.java:923)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):   Caused by: com.google.android.exoplayer2.upstream.DataSourceException
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.open(DefaultHttpDataSource.java:402)
11-21 20:03:31.485 E/ExoPlayerImplInternal(28020):       ... 7 more
11-21 20:03:31.486 E/VideoManager(28020): ***** Got error from player
11-21 20:03:31.486 E/PlaybackController(28020): Playback error - Failed to load video
11-21 20:03:31.491 I/PlaybackController(28020): Player error encountered - retrying
11-21 20:03:31.491 D/PlaybackController(28020): stop called at 0
11-21 20:03:31.642 I/jellyfin-apiclient(28020): ReportPlaybackStopped: Item 5655da47-45cf-7366-b05b-1788c1f5e50c, Ticks: 0
11-21 20:03:31.642 D/jellyfin-apiclient(28020): Adding request to queue: https://<domain>/Sessions/Playing/Stopped
11-21 20:03:31.643 D/PlaybackController(28020): Play called from state: IDLE with pos: 0 and sub index: null
11-21 20:03:31.644 D/PlaybackController(28020): Max bitrate is: 120000000
11-21 20:03:31.646 I/PlaybackController(28020): *** Using extended Exoplayer profile options
11-21 20:03:31.647 D/jellyfin-apiclient(28020): Adding request to queue: https://<domain>/Items/5655da47-45cf-7366-b05b-1788c1f5e50c/PlaybackInfo?format=json
11-21 20:03:31.710 I/jellyfin-apiclient(28020): Response received from: https://<domain>/Items/5655da47-45cf-7366-b05b-1788c1f5e50c/PlaybackInfo?format=json
11-21 20:03:31.712 I/PlaybackController(28020): VLC would direct stream
11-21 20:03:31.716 D/jellyfin-apiclient(28020): Adding request to queue: https://<domain>/Items/5655da47-45cf-7366-b05b-1788c1f5e50c/PlaybackInfo?format=json
11-21 20:03:31.717 I/jellyfin-apiclient(28020): Response received from: https://<domain>/Sessions/Playing/Stopped
11-21 20:03:31.780 I/jellyfin-apiclient(28020): Response received from: https://<domain>/Items/5655da47-45cf-7366-b05b-1788c1f5e50c/PlaybackInfo?format=json
11-21 20:03:31.782 I/PlaybackController(28020): Internal player would direct stream
11-21 20:03:31.782 I/PlaybackController(28020): Explicit deinterlacing will NOT be used
11-21 20:03:31.783 I/PlaybackController(28020): User preferred player is: EXOPLAYER
11-21 20:03:31.783 I/PlaybackController(28020): Will use internal player
11-21 20:03:31.788 D/PlaybackController(28020): default audio index set to 1
11-21 20:03:31.789 D/PlaybackController(28020): default audio index set to 1 remote default 1
11-21 20:03:31.789 D/PlaybackController(28020): default sub index set to -1 remote default -1
11-21 20:03:31.789 I/PlaybackController(28020): Playing back in native mode.
11-21 20:03:31.790 D/PlaybackController(28020): trying to find display mode for video: 3840x2160@23.976025
11-21 20:03:31.790 D/PlaybackController(28020): considering display mode: 1 - 1280x720@29.970030
11-21 20:03:31.790 D/PlaybackController(28020): considering display mode: 2 - 1280x720@23.976025
11-21 20:03:31.790 D/PlaybackController(28020): considering display mode: 3 - 1920x1080@29.970030
11-21 20:03:31.790 D/PlaybackController(28020): considering display mode: 4 - 1920x1080@23.976025
11-21 20:03:31.791 D/PlaybackController(28020): considering display mode: 5 - 1280x720@59.940060
11-21 20:03:31.791 D/PlaybackController(28020): considering display mode: 6 - 1920x1080@59.940060
11-21 20:03:31.791 D/PlaybackController(28020): considering display mode: 7 - 3840x2160@29.970030
11-21 20:03:31.791 D/PlaybackController(28020): considering display mode: 8 - 3840x2160@23.976025
11-21 20:03:31.791 D/PlaybackController(28020): qualifying display mode: 8 - 3840x2160@23.976025
11-21 20:03:31.791 D/PlaybackController(28020): preferring mode: 8 - 3840x2160@23.976025
11-21 20:03:31.792 D/PlaybackController(28020): considering display mode: 9 - 1920x1080@120.000008
11-21 20:03:31.792 D/PlaybackController(28020): considering display mode: 10 - 720x480@60.000004
11-21 20:03:31.792 D/PlaybackController(28020): considering display mode: 11 - 1280x720@30.000002
11-21 20:03:31.792 D/PlaybackController(28020): considering display mode: 12 - 1280x720@24.000002
11-21 20:03:31.793 D/PlaybackController(28020): considering display mode: 13 - 1920x1080@30.000002
11-21 20:03:31.793 D/PlaybackController(28020): considering display mode: 14 - 1920x1080@24.000002
11-21 20:03:31.793 D/PlaybackController(28020): considering display mode: 15 - 1280x720@50.000000
11-21 20:03:31.793 D/PlaybackController(28020): considering display mode: 16 - 1920x1080@50.000000
11-21 20:03:31.793 D/PlaybackController(28020): considering display mode: 17 - 3840x2160@30.000002
11-21 20:03:31.793 D/PlaybackController(28020): considering display mode: 18 - 3840x2160@25.000000
11-21 20:03:31.794 D/PlaybackController(28020): considering display mode: 19 - 3840x2160@24.000002
11-21 20:03:31.794 D/PlaybackController(28020): considering display mode: 20 - 3840x2160@50.000000
11-21 20:03:31.794 D/PlaybackController(28020): considering display mode: 21 - 1280x720@60.000004
11-21 20:03:31.794 D/PlaybackController(28020): considering display mode: 22 - 1920x1080@60.000004
11-21 20:03:31.794 D/PlaybackController(28020): considering display mode: 23 - 3840x2160@60.000004
11-21 20:03:31.794 D/PlaybackController(28020): considering display mode: 24 - 3840x2160@59.940060
11-21 20:03:31.795 I/PlaybackController(28020): *** Best refresh mode is: 8 - 3840x2160/23.976025
11-21 20:03:31.795 I/PlaybackController(28020): Display is already in best mode
11-21 20:03:31.795 D/VideoManager(28020): Setting playback speed: 1.000000
11-21 20:03:31.801 I/VideoManager(28020): Video path set to: https://<domain>/Videos/5655da47-45cf-7366-b05b-1788c1f5e50c/stream.mkv?api_key=<snip>&DeviceId=<snip>&MediaSourceId=5655da4745cf7366b05b1788c1f5e50c&Tag=51360ade4b96a7943bfaa6b2e992fdd5&Static=true
11-21 20:03:31.803 D/VideoManager(28020): Caught player timeline change - reason: PLAYLIST_CHANGED
11-21 20:03:31.804 D/VideoManager(28020): Player is buffering
11-21 20:03:31.804 D/jellyfin-apiclient(28020): Adding request to queue: https://<domain>/Sessions/Playing
11-21 20:03:31.805 I/ReportingHelper(28020): Playback of Trace Decay started.
11-21 20:03:31.829 D/VideoManager(28020): Caught player timeline change - reason: SOURCE_UPDATE
11-21 20:03:31.887 I/jellyfin-apiclient(28020): Response received from: https://<domain>/Sessions/Playing
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020): Playback error
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):   com.google.android.exoplayer2.ExoPlaybackException: Source error
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.ExoPlayerImplInternal.handleIoException(ExoPlayerImplInternal.java:632)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.ExoPlayerImplInternal.handleMessage(ExoPlayerImplInternal.java:604)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at android.os.Handler.dispatchMessage(Handler.java:102)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at android.os.Looper.loop(Looper.java:223)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at android.os.HandlerThread.run(HandlerThread.java:67)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):   Caused by: com.google.android.exoplayer2.upstream.HttpDataSource$InvalidResponseCodeException: Response code: 416
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.open(DefaultHttpDataSource.java:404)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.upstream.DefaultDataSource.open(DefaultDataSource.java:258)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.upstream.StatsDataSource.open(StatsDataSource.java:84)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.source.ProgressiveMediaPeriod$ExtractingLoadable.load(ProgressiveMediaPeriod.java:1005)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.upstream.Loader$LoadTask.run(Loader.java:412)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at java.lang.Thread.run(Thread.java:923)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):   Caused by: com.google.android.exoplayer2.upstream.DataSourceException
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       at com.google.android.exoplayer2.upstream.DefaultHttpDataSource.open(DefaultHttpDataSource.java:402)
11-21 20:03:31.945 E/ExoPlayerImplInternal(28020):       ... 7 more
11-21 20:03:31.946 E/VideoManager(28020): ***** Got error from player
11-21 20:03:31.946 E/PlaybackController(28020): Playback error - Failed to load video
11-21 20:03:31.961 D/ACRA    (28020): onActivityPaused class org.jellyfin.androidtv.ui.playback.PlaybackOverlayActivity
11-21 20:03:31.994 D/ACRA    (28020): onActivityStarted class org.jellyfin.androidtv.ui.browsing.MainActivity
11-21 20:03:31.996 D/ACRA    (28020): onActivityResumed class org.jellyfin.androidtv.ui.browsing.MainActivity
11-21 20:03:31.996 D/BackgroundServiceFragment(28020): Restoring active backgrounds
11-21 20:03:31.997 D/BackgroundServiceFragment(28020): Restoring background drawable
11-21 20:03:32.038 E/libc    (28020): Access denied finding property "ro.vendor.sys.NV_ASYNC"
11-21 20:03:32.038 E/libc    (28020): Access denied finding property "ro.vendor.sys.NV_LOGFPS"
11-21 20:03:32.038 E/libc    (28020): Access denied finding property "ro.vendor.sys.NV_PROCESS_NAME"
11-21 20:03:32.038 E/libc    (28020): Access denied finding property "ro.vendor.sys.NV_FPSDYNAMIC"
11-21 20:03:32.132 W/AudioSystem(28020): ioConfigChanged() modifying unknown output! 182
11-21 20:03:32.133 W/AudioSystem(28020): ioConfigChanged() modifying unknown output! 981
11-21 20:03:32.372 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(28020): Receiving (raw) message {"MessageType":"UserDataChanged","MessageId":"b9d1dd24750f44f6a7d5700ffad13de7","Data":{"UserId":"3866ab99e52044e489a2d7ebcb9b9764","UserDataList":[{"PlaybackPositionTicks":0,"PlayCount":7,"IsFavorite":false,"LastPlayedDate":"2022-11-21T20:03:32.269572Z","Played":false,"Key":"296762001008","ItemId":"5655da4745cf7366b05b1788c1f5e50c"},{"UnplayedItemCount":3,"PlaybackPositionTicks":0,"PlayCount":0,"IsFavorite":false,"Played":false,"Key":"296762001","ItemId":"35835b11a2768eb3bd1423485dbefe6a"}]}}
11-21 20:03:32.465 W/AudioSystem(28020): ioConfigChanged() modifying unknown output! 190
11-21 20:03:32.466 W/AudioSystem(28020): ioConfigChanged() modifying unknown output! 1029
11-21 20:03:32.491 E/Surface (28020): freeAllBuffers: 1 buffers were freed while being dequeued!
11-21 20:03:32.497 D/ACRA    (28020): onActivityStopped class org.jellyfin.androidtv.ui.playback.PlaybackOverlayActivity
11-21 20:03:32.498 D/CustomPlaybackOverlayFragment(28020): Stopping!
11-21 20:03:32.498 D/CustomPlaybackOverlayFragment(28020): this fragment belongs to the current session, ending it
11-21 20:03:32.498 D/PlaybackController(28020): stop called at 0
11-21 20:03:32.649 I/jellyfin-apiclient(28020): ReportPlaybackStopped: Item 5655da47-45cf-7366-b05b-1788c1f5e50c, Ticks: 0
11-21 20:03:32.649 D/jellyfin-apiclient(28020): Adding request to queue: https://<domain>/Sessions/Playing/Stopped
11-21 20:03:32.653 I/ExoPlayerImpl(28020): Release 3c1157a [ExoPlayerLib/2.18.1] [mdarcy, SHIELD Android TV, NVIDIA, 30] [goog.exo.core, goog.exo.ui, goog.exo.exoplayer, goog.exo.decoder, goog.exo.datasource, goog.exo.extractor]
11-21 20:03:32.655 D/ACRA    (28020): onActivityDestroyed class org.jellyfin.androidtv.ui.playback.PlaybackOverlayActivity
11-21 20:03:32.668 D/AndroidRuntime(28020): Shutting down VM
--------- beginning of crash
11-21 20:03:32.668 E/AndroidRuntime(28020): FATAL EXCEPTION: main
11-21 20:03:32.668 E/AndroidRuntime(28020): Process: org.jellyfin.androidtv, PID: 28020
11-21 20:03:32.668 E/AndroidRuntime(28020): java.lang.IllegalStateException: can't get VLCObject instance
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at org.videolan.libvlc.MediaPlayer.nativeSetAudioOutputDevice(Native Method)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at org.videolan.libvlc.MediaPlayer.setAudioOutputDeviceInternal(MediaPlayer.java:995)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at org.videolan.libvlc.MediaPlayer.updateAudioOutputDevice(MediaPlayer.java:485)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at org.videolan.libvlc.MediaPlayer.access$600(MediaPlayer.java:58)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at org.videolan.libvlc.MediaPlayer$3.onAudioDevicesChanged(MediaPlayer.java:561)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at org.videolan.libvlc.MediaPlayer$3.onAudioDevicesRemoved(MediaPlayer.java:585)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at android.media.AudioManager$NativeEventHandlerDelegate$1.handleMessage(AudioManager.java:6270)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at android.os.Handler.dispatchMessage(Handler.java:106)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at android.os.Looper.loop(Looper.java:223)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at android.app.ActivityThread.main(ActivityThread.java:7668)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at java.lang.reflect.Method.invoke(Native Method)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
11-21 20:03:32.668 E/AndroidRuntime(28020): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
11-21 20:03:32.669 E/ACRA    (28020): ACRA caught a IllegalStateException for org.jellyfin.androidtv
11-21 20:03:32.669 E/ACRA    (28020): java.lang.IllegalStateException: can't get VLCObject instance
11-21 20:03:32.669 E/ACRA    (28020): 	at org.videolan.libvlc.MediaPlayer.nativeSetAudioOutputDevice(Native Method)
11-21 20:03:32.669 E/ACRA    (28020): 	at org.videolan.libvlc.MediaPlayer.setAudioOutputDeviceInternal(MediaPlayer.java:995)
11-21 20:03:32.669 E/ACRA    (28020): 	at org.videolan.libvlc.MediaPlayer.updateAudioOutputDevice(MediaPlayer.java:485)
11-21 20:03:32.669 E/ACRA    (28020): 	at org.videolan.libvlc.MediaPlayer.access$600(MediaPlayer.java:58)
11-21 20:03:32.669 E/ACRA    (28020): 	at org.videolan.libvlc.MediaPlayer$3.onAudioDevicesChanged(MediaPlayer.java:561)
11-21 20:03:32.669 E/ACRA    (28020): 	at org.videolan.libvlc.MediaPlayer$3.onAudioDevicesRemoved(MediaPlayer.java:585)
11-21 20:03:32.669 E/ACRA    (28020): 	at android.media.AudioManager$NativeEventHandlerDelegate$1.handleMessage(AudioManager.java:6270)
11-21 20:03:32.669 E/ACRA    (28020): 	at android.os.Handler.dispatchMessage(Handler.java:106)
11-21 20:03:32.669 E/ACRA    (28020): 	at android.os.Looper.loop(Looper.java:223)
11-21 20:03:32.669 E/ACRA    (28020): 	at android.app.ActivityThread.main(ActivityThread.java:7668)
11-21 20:03:32.669 E/ACRA    (28020): 	at java.lang.reflect.Method.invoke(Native Method)
11-21 20:03:32.669 E/ACRA    (28020): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
11-21 20:03:32.669 E/ACRA    (28020): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
11-21 20:03:32.669 D/ACRA    (28020): Building report
11-21 20:03:32.669 D/ACRA    (28020): Starting collectors with priority FIRST
11-21 20:03:32.670 D/ACRA    (28020): Calling collector org.acra.collector.DropBoxCollector
11-21 20:03:32.670 D/ACRA    (28020): Collector org.acra.collector.DropBoxCollector completed
11-21 20:03:32.670 D/ACRA    (28020): Calling collector org.acra.collector.LogCatCollector
11-21 20:03:32.673 D/ACRA    (28020): Retrieving logcat output (buffer:default)...
11-21 20:03:32.673 D/ACRA    (28020): Calling collector org.acra.collector.StacktraceCollector
11-21 20:03:32.674 D/ACRA    (28020): Collector org.acra.collector.StacktraceCollector completed

```  

### App information

***App version***: `0.15.0-beta.2` (`150002`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"darcy","BOOTLOADER":"32.00.2019.50-t210-69ebfcbe","BRAND":"NVIDIA","CPU_ABI":"arm64-v8a","CPU_ABI2":"","DEVICE":"mdarcy","DISPLAY":"RQ1A.210105.003.7825230_3167.5736","FINGERPRINT":"NVIDIA\/mdarcy\/mdarcy:11\/RQ1A.210105.003\/7825230_3167.5736:user\/release-keys","HARDWARE":"darcy","HOST":"mobile-u64-5308-d7000","ID":"RQ1A.210105.003","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"NVIDIA","MODEL":"SHIELD Android TV","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"mdarcy","RADIO":"unknown","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":["arm64-v8a"],"SUPPORTED_ABIS":["arm64-v8a","armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1665975826000,"TYPE":"user","UNKNOWN":"unknown","USER":"NVIDIA","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","INCREMENTAL":"7825230_3167.5736","PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"11","RELEASE_OR_CODENAME":"11","SDK":"30","SDK_INT":30,"SECURITY_PATCH":"2022-04-05"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":150002,"VERSION_NAME":"0.15.0-beta.2"}
```  

### Device information

***Android version***: `11`  
***Device brand***: `NVIDIA`  
***Device product***: `mdarcy`  
***Device model***: `SHIELD Android TV`  

### Crash information

***Start time***: `2022-11-21T20:03:18.422+00:00`  
***Crash time***: `2022-11-21T20:03:32.691+00:00`  



### Application version

0.15.0-beta2

### Where did you install the app from?

Google Play

### Device information

Nvidia Shield 2019 Pro

### Android version

Android 11

### Jellyfin server version

10.8.7

--- a/app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java
@@ -428,7 +428,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
             } else if (mGuideVisible) {
                 hideGuide();
             } else if (!requireActivity().isFinishing()) {
-                requireActivity().finish();
+                onStop();
             }
         }
     };

